receiver/datadogreceiver/receiver.go

```diff
@@ -91,6 +91,7 @@ func (ddr *datadogReceiver) handleTraces(w http.ResponseWriter, req *http.Reques
 	if err != nil {
 		http.Error(w, "Unable to unmarshal reqs", http.StatusInternalServerError)
 		ddr.params.Logger.Error("Unable to unmarshal reqs")
+		return
```

Comment:
Do you think it's feasible to add a test for this?

Reply:
Yeah, sorry it too me a bit to figure out how best to test it.